public/app/features/dashboard-scene/scene/PanelControls.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { ReactElement } from 'react';
+
+import { SceneObjectState, SceneObjectBase, SceneObject } from '@grafana/scenes';
+import { Box, Stack } from '@grafana/ui';
+
+interface PanelControlsState extends SceneObjectState {
+  otherControls: ReactElement[];
+  timeControls: SceneObject[];
+}
+
+export class PanelControls extends SceneObjectBase<PanelControlsState> {
+  static Component = PanelControlsRenderer;
+}
+
+function PanelControlsRenderer({ model }: { model: PanelControls }) {
+  const { otherControls, timeControls } = model.useState();
+
+  return (
+    <Stack
+      grow={1}
+      direction={{
+        md: 'row',
+        xs: 'column',
+      }}
+    >
+      <Box grow={1} />
+      {otherControls.map((control, index) => React.cloneElement(control, { key: index }))}
```

Comment:
Needed this to avoid the unique ids warning. Wasn't sure if `otherControls` was a good name for a prop, and if this should be a `SceneObject`. Maybe it would be an overkill because the only remaining control that needs to be implemented is Fill/Actual toggle which is an `InlineSwitch` component